backend/src/models/syncTables.ts

```diff
@@ -15,13 +15,13 @@ const syncTables = async () => {
       ServiceModel.sync({ alter: true }),
       ComboModel.sync({ alter: true }),
       EmployeeModel.sync({ alter: true }),
-      AppointmentModel.sync({ alter: true }),
+      Customer.sync({ alter: true }),
     ]);
+    await AppointmentModel.sync({ alter: true });
     await ServiceComboModel.sync({ alter: true });
     await EmployeeServiceModel.sync({ alter: true });
     await AppointmentEmployeeModel.sync({ alter: true });
     await AppointmentServiceModel.sync({ alter: true });
-    await Customer.sync({ alter: true });
     await Appointment.sync({ alter: true });
```

Comment:
This model will not be used to store Appointment instances. All code (e.g. `appointment.model.ts`) related to this model are effectively *dead code* and should therefore be removed.